src/chain-wallet-libs/bindings/wallet-wasm-js/js/index.d.ts

```diff
@@ -58,27 +62,44 @@ export class Proposal {
 export class Vote {
   proposal: Proposal;
   choice: number;
+  purpose: VotingPurpose;
+  expiration?: BlockDate;
+  spendingCounter?: number;
+  spendingCounterLane?: number;
 
   /**
    * Vote constructor
    *
    * @param {Proposal} proposal
-   * @param {number} choice choosen vote plan option
+   * @param {number} choice choosen vote plan option.
+   * @param {VotingPurpose} purpose The voting purpose being voted on (Currently not used actually, can pass anything).
```

Comment:
Additionally, we need to be consistent with comments TS/JS. We jump between starting comments from capital letters to regular: `choosen vote plan option.` - `The voting purpose being ...`. I'd propose always starting comments with a capital letter.

Reply:
fixed